backend/clubs/views.py

```diff
@@ -2352,6 +2404,45 @@ def get_used_variables(self):
         return sorted(self._called)
 
 
+class OptionListView(APIView):
```

Comment:
What does this do?

Reply:
Penn Labs has created a package called [django-runtime-options](https://pypi.org/project/django-runtime-options/), which allows you to set options in the Django admin. This is a bit cleaner than having to make a code change and PR every time we want to tweak a setting, like the status of fair registration.

Since we're now loading this information from a model and appropriately calculating some of the fields, I override the options endpoint to return the correct value for the fair options. An alternative solution would be to have another endpoint to return fair specific information, but since this endpoint would be called every time a page is loaded on Penn Clubs, I figure that it would be easier to consolidate all of the site wide variables into one endpoint.